lib/rspec/rails/matchers/have_http_status.rb

```diff
@@ -255,12 +260,11 @@ def initialize(type)
             @invalid_response = nil
           end
 
-          # @return [Boolean] `true` if Rack's associated numeric HTTP code matched
-          #   the `response` code
+          # @return [Boolean] value of response status method
```

Comment:
I'd like to see the original doc restored, with maybe an additional " or the named response status matches"